ManageCoursesUi.Tests/ViewModelHelperTests.cs

```diff
@@ -52,15 +52,15 @@ class ViewModelHelperTests
         [TestCase("s", "s", "d", "s", "Not running")]
         [TestCase("s", "s", "s", "d", "Not running")]
         [TestCase("S", "", "", "S", "Not running")]
-        [TestCase("", "", "", "", "")]
+        [TestCase("", "", "", "", "Not running")]
         public void TestGetCourseStatus(string status1, string status2, string status3, string status4, string expectedResult)
         {
             var schools = new List<CourseSite>
             {
-                new CourseSite {Status = status1},
-                new CourseSite {Status = status2},
-                new CourseSite {Status = status3},
-                new CourseSite {Status = status4}
+                new CourseSite {Publish = "Y", Status = status1},
+                new CourseSite {Publish = "Y", Status = status2},
+                new CourseSite {Publish = "Y", Status = status3},
+                new CourseSite {Publish = "Y", Status = status4}
```

Comment:
Does there need to be a test case where Publish is set to N?

Reply:
To need, or not to need... 😄

I briefly considered doing this, but given that it's on the part of the stack that's going away, did not spend the requisite effort.

We're not testing that bit, but then again, I don't think we need to...

Reply:
Couldn't we just add another column to the test cases here for the Publish value?

I know we're getting rid of this "soon", but if it's fast to do I'd be tempted to do it as proof that whatever is going out here, now, actually works.

Reply:
I'll see if I can do this quickly

Reply:
Maybe just for R and N that's about it

Clone and amend the test and target those test case to, this test is large enough already

Reply:
done, pushed to this branch